FAQ.md

```diff
@@ -58,15 +58,15 @@ In terms of compilers, these are what we currently use for each release:
 
 | Version | OS      | Compiler |
 |---------|---------|----------|
-| JDK8    | Linux   | GCC 4.8 (HotSpot) GCC 7.6 (OpenJ9)                        |
```

Comment:
I personally think this looks nicer with the final vertical bars lined up (as they are in the rest of the table)